status/flowcells.py

```diff
@@ -289,4 +289,16 @@ def post(self, flowcell):
                 self.set_header("Content-type", "application/json")
                 self.finish(json.dumps(links))
 
+#Functions
+def get_container_from_id(flowcell):
+    if flowcell[7:].startswith('00000000'):
```

Comment:
I am pretty sure there isn't :-/ Unless you have a mapping instrument ID <--> MiSeq, which is not as reliable as checking for this `'00000000'` string, which will always be on a MiSeq run